single.php

```diff
@@ -16,14 +16,6 @@
 
 			get_template_part( 'content', 'single' );
 
-			/**
-			 * Functions hooked in to storefront_single_post_after action
-			 *
-			 * @hooked storefront_post_nav         - 10
-			 * @hooked storefront_display_comments - 20
-			 */
-			do_action( 'storefront_single_post_after' );
```

Comment:
I think we should leave the actions where they were, but then add _new_ actions inside the `article`. `storefront_single_post_top`, `storefront_single_post_bottom`. Then hook those functions into the new actions instead.